test/lg-logic.js

```diff
@@ -9,7 +9,7 @@ describe('calcValue works as it should', () => {
   it('gutter, rounder, and unit ✅', () => {
     var testCase = lgLogic.calcValue('1/3', '30px', 100, 'vw');
 
-    var expectedResult = 'calc(100vw * 1/3 - (30px - 30px * 1/3))';
```

Comment:
Could we pull off the trailing zeros within the `calcValue` function? CSS Clean looks to be stripping them out for us in the tests, but without it, we're leaving an optimization on the table. What're five zeroes though? Nothing worth splitting hairs over *but* if we're already doing optimizations it would be cool to take it all the way. **not blocking merge**